src/test/java/com/example/rets_api/service/PropertyServiceTest.java

```diff
@@ -41,7 +44,7 @@ public class PropertyServiceTest {
     @Test
     public void shouldCreateProperty_createProperty() {
         //Creating objects
-        PropertyDTO propertyDTORequest = DtoUtilsTest.createPropertyDTOWithBasicFields();
+        PropertyDTO propertyDTORequest = createPropertyDTOWithBasicFields();
```

Comment:
Please change the name of this method createPropertyDTOWithBasicFields inside DtoUtilsTest to createPropertyDTOWithAllFields and create a separated one just with basic fields and use the name createPropertyDTOWithBasicFields. Now the method createPropertyDTOWithBasicFields is creating a property with all fields (from line 32 to 40) not only the basic fields (from line 14 to 31)